app/core/auth-options.ts

```diff
@@ -378,8 +381,14 @@ export const authOptions: AuthOptions = {
   secret: AUTH_SECRET,
   callbacks: {
     async signIn({ user, account, profile }) {
-      const { given_name, family_name, email } = profile as KeycloakProfile;
+      const { given_name, family_name, email, idirGuid } = profile as KeycloakProfile;
+
+      if (!idirGuid) {
+        console.warn(`Login blocked: Missing idirGuid for user ${user?.email}`);
+        return false;
```

Comment:
Is there logging setup for this app that will capture this console.warn message?